ui/lib/open-api-explorer/addon/components/swagger-ui.js

```diff
@@ -26,18 +26,24 @@ export default class SwaggerUiComponent extends Component {
     return {
       fn: {
         opsFilter: (taggedOps, phrase) => {
-          // map over the options and filter out operations where the path doesn't match what's typed
-          return (
-            taggedOps
-              .map((tagObj) => {
-                const operations = tagObj.operations.filter((operationObj) => {
```

Comment:
this line was failing because the type of `tagObj` changed from a `Map` to a `List`, and we couldn't call `.operations` from a `List`.

![image (6)](https://github.com/hashicorp/vault/assets/903288/52059e25-3eb0-48cb-be5a-7414abf0250a)


![image (7)](https://github.com/hashicorp/vault/assets/903288/a0bc706b-8a9f-4d8a-84d8-03347f7c8f4c)